crates/bevy_core_pipeline/src/blit/mod.rs

```diff
@@ -19,18 +19,14 @@ impl Plugin for BlitPlugin {
     fn build(&self, app: &mut App) {
         embedded_asset!(app, "blit.wgsl");
 
-        if let Some(render_app) = app.get_sub_app_mut(RenderApp) {
-            render_app.allow_ambiguous_resource::<SpecializedRenderPipelines<BlitPipeline>>();
-        }
-    }
-
-    fn finish(&self, app: &mut App) {
         let Some(render_app) = app.get_sub_app_mut(RenderApp) else {
             return;
         };
+
         render_app
-            .init_resource::<BlitPipeline>()
-            .init_resource::<SpecializedRenderPipelines<BlitPipeline>>();
+            .allow_ambiguous_resource::<SpecializedRenderPipelines<BlitPipeline>>()
```

Comment:
i feel like a lot of "just data" resources like this and configs etc should be `allow_ambiguous_resource`. Immutable resources pleaaaaaasseeeeeeeeeeeeee @alice-i-cecile